src/System.Net.Http/src/netcore50/System/Net/HttpClientHandler.cs

```diff
@@ -522,6 +469,82 @@ private void ApplyDecompressionSetting(HttpRequestMessage request, Decompression
                 _rtFilter.ClientCertificate = clientCertificates[0];
             }
         }
+
+        private RTPasswordCredential RTPasswordCredentialFromICredentials(ICredentials creds)
+        {
+            // The WinRT PasswordCredential object does not have a special credentials value for "default credentials".
+            // In general, the UWP HTTP platform automatically manages sending default credentials, if no explicit
+            // credential was specificed, based on if the app has EnterpriseAuthentication capability and if the endpoint
+            // is listed in an intranet zone.
+            //
+            // A WinRT PasswordCredential object that is either null or created with the default constructor (i.e. with
+            // empty values for username and password) indicates that there is no explicit credential. And that means
+            // that the default logged-on credentials might be sent to the endpoint.
+            //
+            // There is currently no WinRT API to turn off sending default credentials other than the capability
```

Comment:
>so basically we cannot implement DefaultCredentials=false because it needs to send at least some fake, non-null user-name.

Correct.  And although I thought about doing that (creating a "fake" username, i.e. guid-string), it would actually break app-compat behavior with prior UWP .NET Core (1.0, 1.1, etc.).  It is better to keep the behavior and document the platform differences.